smartcar/requester.py

```diff
@@ -58,7 +61,7 @@ def call(method, url, **kwargs):
             response.raise_for_status()
 
     except Exception as e:
-        if isinstance(e, E.SmartcarException):
+        if isinstance(e, E.SmartcarException) or isinstance(e, E.SmartcarExceptionV2):
```

Comment:
If we do what we did in Java, SmartcarExceptionV2 should inherit from SmartcarException and then we only have to do one instance check (same with our clients)